test/es6/rlexe.xml

```diff
@@ -1331,4 +1331,12 @@
     <compile-flags>-force:deferparse -args summary -endargs</compile-flags>
   </default>
 </test>
+<test>
+  <default>
+    <files>bug_issue_1496.js</files>
+    <compile-flags>-mic:1 -maxsimplejitruncount:2</compile-flags>
```

Comment:
No harm - it's value vs. (a little extra) run time.
Let's get 2nd opinion.
@ianwjhalliday, what do you think?